core/src/main/java/com/netflix/conductor/core/reconciliation/WorkflowSweeper.java

```diff
@@ -100,10 +100,13 @@ public void sweep(String workflowId) {
         long workflowOffsetTimeout =
                 workflowOffsetWithJitter(properties.getWorkflowOffsetTimeout().getSeconds());
         if (workflow != null) {
-            long startTime = Instant.now().toEpochMilli();
+            // long startTime = Instant.now().toEpochMilli();
             unack(workflow, workflowOffsetTimeout);
-            long endTime = Instant.now().toEpochMilli();
-            Monitors.recordUnackTime(workflow.getWorkflowName(), endTime - startTime);
+            // long endTime = Instant.now().toEpochMilli();
+            // This metrics(recordUnackTime) has workflowName in our prod account we have more number of workflows.
+            // So we may get cardinality problems in haystack due to this.
+            // Disabling this temporarily.
+            // Monitors.recordUnackTime(workflow.getWorkflowName(), endTime - startTime);
```

Comment:
Can we replace this with a log entry ?

Reply:
we need to add bb logs for sweeper and reconciler with logs standardisation task planning to add that log

ref: https://docs.google.com/spreadsheets/d/1GVa5Vwk_NW6fY5y9MtnaH0nIv06VcynldO87CNFXjqs/edit?gid=0#gid=0